tools/ClassGen/InstrGen.cpp

```diff
@@ -671,7 +671,8 @@ int main(int argc, char **argv) {
   BB.newInstr("Touch")
       .addOperand("Dest", OperandKind::Out)
       .dataParallel()
-      .autoVerify(VerifyKind::NoVerify);
+      .autoVerify(VerifyKind::NoVerify)
```

Comment:
Should InstrGen and NodeGen changes be moved into a separate PR? These changes seem to be pretty independent of the rest of this PR.

Reply:
I use the TouchNode in this PR so I'm not sure why I should put this in another PR ...